envs/tensorflow-s390x-extra-deps-env.yaml

```diff
@@ -0,0 +1,5 @@
+packages:
+{% if s390x %}
```

Comment:
Sure, we can move the s390x condition during import inside `tensorflow-env.yaml`. Only reason for adding it here to avoid generating builds for other platform in case of any manual error.

Reply:
Fine. LGTM